yarn.lock

```diff
@@ -4988,6 +5027,28 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@videojs/vhs-utils@npm:^3.0.5":
+  version: 3.0.5
+  resolution: "@videojs/vhs-utils@npm:3.0.5"
+  dependencies:
+    "@babel/runtime": "npm:^7.12.5"
+    global: "npm:^4.4.0"
+    url-toolkit: "npm:^2.2.1"
+  checksum: 10/da1dbbd75dc0f9971a9efb7301b886b08dd3eb9734f268cd5338dfd97a9a03d44a1d8103173193bbe0558a730858d5ff49a8e0bbee0415d976e63574081783e6
+  languageName: node
+  linkType: hard
+
+"@videojs/vhs-utils@npm:^4.0.0":
+  version: 4.1.0
+  resolution: "@videojs/vhs-utils@npm:4.1.0"
+  dependencies:
+    "@babel/runtime": "npm:^7.12.5"
+    global: "npm:^4.4.0"
+    url-toolkit: "npm:^2.2.1"
+  checksum: 10/6cf55913e17e585d009fd0c77a225fb7e4e561bb14daaf978929e8a5e931629ff675d835e6347165940b9f7149aa8d6be8dcfa6563ebdf9a28c9b9cfde09e5b6
+  languageName: node
+  linkType: hard
```

Comment:
There's 4 versions of this package! Let's try to cut down. We don't actually use `@videojs/vhs-utils` directly, yet at least in develop it is listed as a direct dependency. Please check it out. See if removing and deduping helps.